llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/VectorExt/IR/VectorExtAttrs.td

```diff
@@ -99,5 +99,170 @@ def LayoutAttr : IREEVectorExt_Attr<"Layout",
   }];
 }
 
-#endif // IREE_DIALECT_VECTOREXT_ATTRS
+def LayoutV2Attr : IREEVectorExt_Attr<"LayoutV2",
+      [ DeclareAttrInterfaceMethods<VectorLayoutInterface> ]> {
+  let mnemonic = "layoutv2";
```

Comment:
I would name this `TiledLayout` or something similar so that we don't have to rename it later after writing a bunch of tests.

Reply:
I find Hierarchical too long. Going to use NestedLayout for now.